### PR TITLE
528 - Present Additional Restrictions on the client 

### PR DESCRIPTION
**Pull request by nozomione**
## Issue Number

Closing #528

(This PR is based on the #527 branch.)

## Purpose/Implementation Notes
- Based on the [projects](https://api.staging.scpca.alexslemonade.org/v1/projects) endpoint's `additional_restrictions` text field (`null` or "Research or academic purposes only"): 
   - added "Additional Restrictions" item to bottom of the project details table
   - added "Additional Restrictions"  to `DownloadStart` component, if populated

## Types of changes
- New feature (non-breaking change which adds functionality)

## Functional tests

List out the functional tests you've completed to verify your changes work locally.

## Checklist

- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by nozomione** [comment]
@davidsmejia you were auto-assigned since I added the `client / front-end` label. But I'll remove your temporarily and will add you back later  once I test my implementation with new API updates, thank you!

**Review by davidsmejia — COMMENTED** [review]
Looks mostly good, just want to remove the temp block.

File: client/src/components/ProjectAdditionalRestrictions.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    : text
+
+  if (!textContent) return <Text italic>None</Text>
+
+  return (
+    <Box direction="row" gap="xsmall">
+      {isModal && <Icon name="Info" />}
+      <Text>{textContent}</Text>
+      {/* NOTE: add a link for terms of use */}
+      <Link href="#temp">
```
[the remark this answers is not in the document]
Comment (nozomione) [reply client/src/components/ProjectAdditionalRestrictions.js]: I've replaced `#temp` with the link of Terms of use's  Access to and Use of Content (section 3) da2ec62bc316ea882ad6cc79c7ab56a8d24c0151.

File: client/src/pages/projects/[scpca_id].js [hunk 2]
```diff
@@ -102,6 +103,14 @@ const Project = ({ project }) => {
                         ) : (
                           ''
                         )
+                    },
+                    {
+                      label: 'Additional Restrictions',
+                      value: (
+                        <ProjectAdditionalRestrictions
+                          text={project.additional_restrictions}
```
[the remark this answers is not in the document]
Comment (nozomione) [reply client/src/pages/projects/[scpca_id].js]: I've added the fallback text `Pending` during the processing a1b01ccea8ab14d40177633dc5305bce26d1936d.

**Review by davidsmejia — APPROVED** [review]
LGTM

**Pull request merged**